src/components/TopBar/HamburgerMenu.jsx

```diff
@@ -12,15 +12,18 @@ const HamburgerMenu = createMenu({
   name: 'hamburger',
 
   renderItems({
+    hasExportedRepo,
     hasInstructions,
     isEditingInstructions,
     isExperimental,
     isGistExportInProgress,
     isRepoExportInProgress,
+    isRepoUpdateInProgress,
```

Comment:
I don’t feel terribly strongly about this, but I think `isRepoExportInProgress` could reasonably capture both situations here, since only one could possibly be true at any given time. So, it might be nice to save ourselves a prop.

(I do think it makes sense to keep the underlying representation in the store the same—I would just suggest combined semantics at the selector level)

Reply:
consolidated `isRepoUpdateInProgress` into `isRepoExportInProgress`